tests/unit/button/button_options.js

```diff
@@ -25,135 +25,80 @@ test( "disabled, explicit value", function() {
 	deepEqual( element.prop( "disabled" ), true, "element is not disabled" );
 });
 
-test("disabled, null", function() {
-	expect( 4 );
-	$("#radio01").button({ disabled: null });
-	deepEqual(false, $("#radio01").button("option", "disabled"),
+test( "disabled, null", function() {
```

Comment:
I'm starting to remember all the logic around disabled. This test should explain why we test this, I still can't tell, since the default is `null` anyway.